src/Arcus.Scripting.Tests.Integration/Files/arm-template-object-value (linux).json

```diff
@@ -0,0 +1,3 @@
+﻿{
+   "test": "this is a test value"
```

Comment:
Don't we need testfiles that contain newline chars ?

Reply:
This test file contains the injected value for Linux. The end-result will contain these newline chars (see assertion). That's also the fix of this PR: multiline file to newline chars.